packages/babylon/.flowconfig

```diff
@@ -0,0 +1,11 @@
+[ignore]
+
+[include]
+
+[libs]
+
+[lints]
+
+[options]
+
+[strict]
```

Comment:
I think this file is not needed. Do you run make lint?

Reply:
Yep, it's useless. I will remove it. Of cause I ran make lint.